full_bubble_model.py

```diff
@@ -577,21 +624,14 @@ def solve(cpar, t_int=np.array([0.0, 1.0]), LSODA_timeout=30.0, Radau_timeout=30
         
     error_code = 0
     start = time.time()
-    if not 'P_v' in cpar:
-        cpar.P_v = vapour_pressure(T=cpar.T_inf) # [Pa]
-    if not 'mu_L' in cpar:
-        cpar.mu_L = viscosity(T=cpar.T_inf) # [Pa]
-    if not 'rho_L' in cpar:
-        cpar.rho_L = par.rho_L # [kg/m^3]
-    if not 'c_L' in cpar:
-        cpar.c_L = par.c_L # [m/s]
+    num_sol = None
+    if not check_cpar(cpar):
```

Comment:
solve() can print error message and call stack when a fatal error is encountered

Python can't trace back errors in JIT-ted functions, so remove @jit when hunting for errors)